File: Easy/412. Fizz Buzz/main.py
from typing import List


class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        ans=[]
        for i in range (1 , n+1):

            divisable_by_3= i%3
            divisable_by_5= i%5
            if divisable_by_3==0 and divisable_by_5==0:
                 ans.append("FizzBuzz")
            elif divisable_by_3==0:
                  ans.append("Fizz")
            elif divisable_by_5==0:
                  ans.append("Buzz")
            else:
                ans.append(str(i))
        
        return ans
    
s=Solution()
print(s.fizzBuzz(15))

File: Easy/412. Fizz Buzz/main.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> res(n);
        for (int i = 1; i <= n; i++)
        {
            string s = "";
            if (i % 15 == 0)
                s += "FizzBuzz";
            else if (i % 3 == 0)
                s += "Fizz";
            else if (i % 5 == 0)
                s += "Buzz";
            else
                s += to_string(i);

            res[i - 1] = s;
        }

        return res;
    }

};
int main() {

    std::cout << "Hello" << std::endl;


    Solution s = Solution();
    auto res = s.fizzBuzz(15);
    for (auto&& i : res)
    {
        cout << i << endl;
    }
    return 0;
}